components/bl-alert-component/README.md

```diff
@@ -3,46 +3,64 @@
 Alert is a component of Backendless UI-Builder designer. This allows you to notify the user of some error, danger, information, or success.
 
 The component based on external [alert](https://mui.com/material-ui/react-alert/).
-## Usage
-
-### Styles and Settings
-
-<dl>
-<dt>Message Title</dt>
-<dd>Allows write title for alert.</dd>
-<dt>Message Text</dt>
-<dd>Allows write text for alert</dd>
-<dt>Message Type</dt>
-<dd>Allows select type of alert (error, warning, info, success). Default value 'error'.</dd>
-<dt>Icon Visibility</dt>
-<dd>Allows make alert with or without icon. Default value 'checked'.</dd>
-<dt>Close Button Visibility</dt>
-<dd>Allows make alert with or without close button. Default value 'checked'.</dd>
-<dt>Style Variants</dt>
-<dd>Allows select style for alert (alert root, alert outline, alert filled). Default value 'alert root'.</dd>
-<dt>Closing Duration</dt>
-<dd>Allows to specify speed animation closing for alert. Default value 0.1</dd>
-</dl>
-
-### Event Handlers and Bindable Properties
-
-<dl>
-<dt>Message Title Logic</dt>
-<dd>Allows to specify title for alert in Backendless logic</dd>
-<dt>Message Text Logic</dt>
-<dd>Allows to specify text for alert in Backendless logic</dt>
-<dt>Message Type Logic</dt>
-<dd>Allows to specify type of alert in Backendless logic</dd>
-<dt>Icon Visibility Logic</dt>
-<dd>Allows to specify with or without icon in Backendless logic</dd>
-<dt>Close Button Visibility Logic</dt>
-<dd>Allows to specify with or without close button in Backendless logic</dd>
-<dt>On Close Button Visibility</dt>
-<dd>Triggered when the user click on close button</dd>
-</dl>
-
-### Action
-<dl>
-<dt>Close Button Visibility</dt>
-<dd>An action that close alert</dd>
-</dl>
+
+## Properties
+
+| Property              | Type     | Default Value | Logic                         | Data Binding | UI Setting | Description                                                              |
+|-----------------------|----------|---------------|-------------------------------|--------------|------------|--------------------------------------------------------------------------|
+| messageTitle          | Text     |               | Message Title Logic           | YES          | YES        | Allows writing the title of message.                                     |
+| messageText           | Text     |               | Message Text Logic            | YES          | YES        | Allows writing the text of message.                                      |                                |
```

Comment:
You have external `|` at end of the line.

Also fix types according to PR checklist wiki